ts/types/wallet.ts

```diff
@@ -14,14 +14,15 @@ export type WalletTransaction = {
   id: number;
   cardId: number;
   isoDatetime: string;
-  date: string;
-  time: string;
+  date: string; // it could be the same of date, having date as date.toLocaleDateString() TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
+  time: string; // it could be the same of date, having date as date.toLocaleTimeString() TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
   paymentReason: string;
   recipient: string;
   amount: number;
   currency: string;
-  transactionCost: number;
+  transactionCost: number; // it should be amount + transactionCost TODO: @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
```

Comment:
how can transactionCost be transactionCost + amount?

Reply:
don't know why it was there.  Actually it should be a reference to the following point.
As reported in the screen, it should be preferred to define if provide the total amount to the screen or if it will be calculated  as the sum of the amount and the fee inside the screen code.
 It the meanwhile it is implemented inside the screen code.
As mentioned in the TODOs, it could be improved with the story related to controls.